core/src/main/java/org/frankframework/pipes/PasswordGeneratorPipe.java

```diff
@@ -123,25 +110,21 @@ protected String garbleString(String input) {
 		return resultSb.toString();
 	}
 
-
-	public boolean isUseSecureRandom() {
-		return useSecureRandom;
-	}
-
 	/**
-	 * whether the securerandom algorithm is to be used (slower)
+	 * Whether the secureRandom algorithm is used.
+	 * @deprecated the current implementation always uses SecureRandom. Please remove this attribute from the Configuration.
 	 * @ff.default true
 	 */
 	public void setUseSecureRandom(boolean b) {
```

Comment:
Ik zou hier dan ook een @Deprecated / @ConfigurationWarning verwachten

Reply:
`@deprecated` staat er al. Je bedoelt alleen de `@ConfigurationWarning`? Dat is iig een goed punt en ga ik nog effe toevoegen.

Reply:
De `@Deprecated` annotatie, niet de javadoc tag ;)